files/socialmedia/update.json

```diff
@@ -0,0 +1,8 @@
+{
+	"packageManager": "github",
+	"name": "socialmedia",
+	"repo": "jabranr/socialmedia",
+	"files": {
+		"include": ["socialmedia.js", "socialmedia.min.js"]
```

Comment:
you need to use `basePath` to set the directory to `dist/` because the included files are there https://github.com/jabranr/socialmedia/tree/master/dist